cypress/support/fragments/settings/acquisitionUnits/acquisitionUnits.js

```diff
@@ -61,7 +61,7 @@ export default {
   assignAdmin: () => {
     cy.do([
       findUserButton.click(),
-      userSearchModal.find(searchTextField).fillIn('diku'),
+      userSearchModal.find(searchTextField).fillIn('folio-aqa'),
```

Comment:
Please use more universal approach. Take the Last name[, First name] from the profileDropdown